osf/external/messages/celery_publishers.py

```diff
@@ -0,0 +1,48 @@
+from kombu import Exchange
+from framework.celery_tasks import app as celery_app
+from framework.celery_tasks.handlers import enqueue_task
+
+
+def publish_deactivated_user(user):
+    enqueue_task(
+        _publish_user_status_change.s(
+            body={
+                'action': 'deactivate',
+                'user_uri': user.get_semantic_iri(),
+            },
+        )
+    )
+
+
+def publish_reactivate_user(user):
+    enqueue_task(
+        _publish_user_status_change.s(
+            body={
+                'action': 'reactivate',
+                'user_uri': user.get_semantic_iri(),
+            },
+        )
+    )
+
+
+def publish_merged_user(user):
+    assert user.merged_by, 'User received merge signal, but has no `merged_by` reference.'
+    enqueue_task(
+        _publish_user_status_change.s(
+            body={
+                'action': 'merge',
+                'user_uri': user.get_semantic_iri(),
+                'merged_user_uri': user.merged_by.get_semantic_iri(),
+            },
+        )
+    )
+
+
+@celery_app.task()
```

Comment:
Does this need to be a celery task? Doesn't that mean that this itself is performed async by osf's celery worker?

Reply:
yeah it must be a task to be enqueued, it just needs to be registered.

Reply:
But this isn't something that needs to be enqueued, this is what's doing the enqueuing of the the task, right? Also, agree that this could be flagged off -- likely with both `settings.USE_CELERY` and the new `ENABLE_GV` flag.